gems/smithy/lib/smithy/forge/client.rb

```diff
@@ -32,8 +33,15 @@ def source_files
           e.yield "lib/#{@gem_name}/customizations.rb", render_customizations
           e.yield "lib/#{@gem_name}/types.rb", render_types
           e.yield "lib/#{@gem_name}/errors.rb", render_errors
+          e.yield "lib/#{@gem_name}/endpoint_parameters.rb", render_endpoint_parameters
+          e.yield "lib/#{@gem_name}/endpoint_provider.rb", render_endpoint_provider
+          e.yield "lib/#{@gem_name}/plugins/endpoint.rb", render_endpoint_plugin
+
+          e.yield 'spec/spec_helper.rb', render_spec_helper
+          e.yield 'spec/endpoint_provider_spec.rb', render_endpoint_provider_spec
```

Comment:
I think by convention this should still be spec/gemname/specs

Reply:
Was following what we had in V3, but I can update.